client/state/selectors/get-site-timezone-string.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Return the site setting `timezone_string` value from the state-tree
+ *
+ * @param  {Object}  state - Global state tree
+ * @param  {Number}  siteId - Site ID
+ * @return {?String} site setting timezone
+ */
+export default function getSiteTimezoneString( state, siteId ) {
```

Comment:
We should consider that we have two values for each timezone: `value` and `label`. This selector returns the `value` since we are storing only this into the `site-settings` state tree.
In fact, we aren't consistent in this sense because now we're sometimes showing the value and other times the label of the timezone. We always should show the timezone label to the user.
After these words, I'd suggest use `getSiteTimezoneValue()` since we will have to implement soon a method which should return the timezone label: geSiteTimezoneLabel`.